armory/metrics/poisoning.py

```diff
@@ -46,6 +55,9 @@ class ExplanatoryModel:
     def __init__(
         self,
         explanatory_model,
+        data_modality="image",
+        model_framework="pytorch",
+        activation_layer=None,
```

Comment:
it's unclear what `activation_layer` is supposed to be

Reply:
Added docstrings